tests/FrontendSpec.hs

```diff
@@ -61,15 +60,16 @@ type TestApi =
   :<|> "303"       :> GetH ST
   :<|> "any_other" :> GetH ST
 
-testApi :: (Monad m, MonadError ServantErr m) => ServerT TestApi m
+testApi :: (Monad m, MonadServantErr err m) => ServerT TestApi m
 testApi =
        error "unexpected"
-  :<|> throwError (err303 { errHeaders = ("Location", "/target") : errHeaders err303 })
-  :<|> throwError err500
+  :<|> throwServantErr (err303 { errHeaders = ("Location", "/target") : errHeaders err303 })
+  :<|> throwServantErr err500
 
 testAppAulaExcept :: IO Application
 testAppAulaExcept = do
-    action <- mkRunAction <$> Persistent.Implementation.STM.mkRunPersist
+-- FIXME
+    action <- mkRunAction <$> (ActionEnv <$> Persistent.Implementation.STM.mkRunPersist <*> pure Config.test)
```

Comment:
I would be glad to have a place for reusable test related code. However Config.test is so small that I would not even bother.